lib/regular/regular.mli

```diff
@@ -817,17 +823,27 @@ module Std : sig
       any specific support for the serialization).*)
   module Bytes : sig
 
-
     (** bytes  *)
     type t = Bytes.t [@@deriving bin_io, compare, sexp]
 
     include Container.S0   with type t := t with type elt := char
     include Blit.S         with type t := t
     include Identifiable.S with type t := t
-    module To_string   : Blit.S_distinct with type src := t with type dst := string
     module From_string : Blit.S_distinct with type src := string with type dst := t
 
+    module To_string  : sig
```

Comment:
I would suggest the following way to deprecated three functions out of this module:
```ocaml
module To_string : sig
    include Blit.S_distinct with type src := string and type dst := t
    val blit : (t, t) Blit.blit
    [@@deprecated "[since 2018-11] use the Bytes.blit instead"]

    (* etc *)
```